aqua/MenuMeters/Portfile

```diff
@@ -4,47 +4,39 @@ PortSystem          1.0
 
 PortGroup           github 1.0
 PortGroup           xcode 1.0
-PortGroup           xcodeversion 1.0
 
-github.setup        yujitach MenuMeters 1.9.8
+github.setup        yujitach MenuMeters 2.1.1
 categories          aqua sysutils
+platforms           macosx
 maintainers         {stevenmyint.com:git @myint} openmaintainer
 license             GPL-2
-
-# bundled MenuCracker is only compiled for these archs
-supported_archs     i386 ppc x86_64
+supported_archs     i386 x86_64
 
 description         Set of CPU, memory, disk, and network monitoring tools
 long_description    The MenuMeters monitors are true SystemUIServer plugins     \
                     (also known as Menu Extras). This means they can be reordered   \
                     using command-drag and remember their positions in the menubar  \
                     across logins and restarts.
 
-checksums           rmd160  a93d08d6e4a0bf180171550c97eaf27b970f3786 \
-                    sha256  3c6b3a22f095da6dccbe0f54211d1b6abd26491421a316b8128335000339be5e \
-                    size    234533
-
-patchfiles          patch-MenuMeters.xcodeproj-project.pbxproj.diff
+checksums           rmd160  51722b43b39703d172a9e40a98d8948b42ce903e \
+                    sha256  35b45bf188b1a3b34ea32defa4a05759bcb451c490cef10f88116df239a1b8a6 \
+                    size    1211513
 
 xcode.configuration Release
-xcode.scheme        PrefPane
-destroot.pre_args   -derivedDataPath ./DerivedData
-build.pre_args      -derivedDataPath ./DerivedData
-
-destroot.violate_mtree \
-                    yes
-
-destroot    {
-    xinstall -m 755 -d ${destroot}/Library/PreferencePanes
-    file copy ${worksrcpath}/build/UninstalledProducts/macosx/${name}.prefPane \
-        ${destroot}/Library/PreferencePanes
-}
+xcode.target        "${name} No Sparkle"
+xcode.destroot.settings \
+    CODE_SIGN_IDENTITY=- \
+    CODE_SIGN_STYLE=Manual \
+    ENABLE_HARDENED_RUNTIME=NO
 
-minimum_xcodeversions {14 8.0}
+# Compile and install in one step, for now. See https://trac.macports.org/ticket/57137
+build {}
```

Comment:
I would think that this would break the `+universal` variant. Have you tested that?

Reply:
This does not break universal. I built a fat binary with +universal:

```
 $ file work/destroot/Applications/MacPorts/MenuMeters.app/Contents/MacOS/MenuMeters
work/destroot/Applications/MacPorts/MenuMeters.app/Contents/MacOS/MenuMeters: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|PIE>] [arm64:Mach-O 64-bit arm64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|PIE>]
```